.github/actions/test-khiops-install/action.yml

```diff
@@ -42,14 +125,26 @@ runs:
           echo "::error::Unexpected output in khiops_coclustering scripts"
           false
         fi
-    - name: Check process number
+    - name: Check Khiops coclustering output [Windows]
+      if: runner.os == 'Windows'
+      shell: cmd /C call {0}
+      run: |
+        khiops_coclustering -s
+        for /f %%i in (' khiops_coclustering -v ^| find /c /v "" ') do set "LINE_NUMBER=%%i"
+        if not "%LINE_NUMBER%" == "1" (
+          echo "::error::Unexpected output in khiops_coclustering scripts"
+          exit 1
+        )
+    - name: Check process number [UNIXes]
+      # Test if Khiops has the right number of logical processes
+      # (it's obvious but you never know)
+      if: runner.os != 'Windows'
       shell: bash
       env:
         KHIOPS_PROC_NUMBER: 4
         OMPI_MCA_rmaps_base_oversubscribe: true
       run: |-
-        PROC_NUMBER_LIST=($(khiops -s | grep 'Logical processes on system'))
-        PROC_NUMBER=${PROC_NUMBER_LIST[-1]}
+        PROC_NUMBER=$(khiops -s | grep "Logical processes" | awk '{print $5}')
```

Comment:
`awk '{print $NF}'` seems more portable to me: it's just the last field that we want to test here, isn't it?